grade.py

```diff
@@ -157,6 +158,8 @@ def prompt_grade(self, rubric_item: RubricItem):
         """Prompts the TA for pts/comments"""
         for i, (pts, desc) in enumerate(rubric_item.subitems, 1):
             subitem_code = f"{rubric_item.code}.{i}"
+            if self.grades.is_graded(subitem_code):
```

Comment:
I'm confused, isn't this what the default mode is anyways? pygrader skips things that are already graded if you didnt run with --regrade